services/QuillLMS/engines/evidence/app/views/evidence/research/gen_ai/stem_vaults/show.html.erb

```diff
@@ -9,27 +9,40 @@
 
   <h3>Stem</h3>
   <p><%= simple_format(@stem_vault.stem) %></p>
+  <h3>
 
-  <h3>Optimal Guidelines</h3>
+  <h3>
+    Datasets
+    <%= link_to 'new', new_research_gen_ai_stem_vault_dataset_path(stem_vault_id: @stem_vault), class: 'new-link', target: '_blank', style: 'margin-right: 20px;' %>
+  </h3>
+  <ul>
+    <% @stem_vault.datasets.each do |dataset| %>
+      <li><%= link_to dataset.id, dataset %></li>
+    <% end %>
+  </ul>
+
+  <h3>
+    Optimal Guidelines
+    <%= link_to 'new', new_research_gen_ai_stem_vault_guideline_path(@stem_vault), class: 'new-link', target: '_blank', style: 'margin-right: 20px;' %>
+  </h3>
   <ul>
     <% @stem_vault.guidelines.optimal.each do |guideline| %>
       <li><%= guideline %></li>
     <% end %>
   </ul>
 
-  <h3>Sub-Optimal Guidelines</h3>
+  <h3>
+    Sub-Optimal Guidelines
```

Comment:
I believe `Suboptimal` is [a word](https://www.merriam-webster.com/dictionary/suboptimal#:~:text=%3A%20less%20than%20optimal%20%3A%20not%20at,suboptimal%20conditions%2Fresults%2Fsolutions).

Reply:
Cool.  That will be much easier in code than having to switch between sub-optimal and sub_optimal